audio_service/darwin/Classes/AudioServicePlugin.m

```diff
@@ -114,6 +114,7 @@ - (void)activateCommandCenter {
         [NSNull null],
         [NSNull null],
         commandCenter.changeShuffleModeCommand,
+        commandCenter.changePlaybackRateCommand,
```

Comment:
Even though I added this code, it doesn't seem to do anything, see https://github.com/ryanheise/audio_service/issues/712#issuecomment-860041683

Reply:
I think the array order is out of sync with the enum order. I'll fix it while merging.

Reply:
Actually it looks right, but since it offsets the iOS-specific commands this would be a breaking change to the platform interface.

Although the approach is a bit fragile. I should probably increase the width of the bit field from 32 to 64 bits and move the iOS-specific ones down to some constant offset.

Reply:
Looking at media2's compatibility layer with the compat libs, it looks like it doesn't have anything equivalent to this action bitmap, and just automatically enables all actions. I should probably do that and then I wouldn't need `androidEnableQueue` anymore.

Then in terms of the enum order, I think we can also assume that there will be no new action constants added on the Android (if this approach is indeed not carried over to media2), and so it wouldn't be necessary to introduce some arbitrary constant offset for where the iOS-specific constants start.

All I would do then is ensure that new constants are always added to the end and not inserted. So `setSpeed` would best appear at the end.

Reply:
This whole thing anyways will need to be totally reworked with upcoming controls API, so Android and iOS parts will be more isolated